src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
There are two options:

1. Keep the existing unwind info format that is embedded into x86 GCInfo
2. Design and implement a brand new unwind info format for x86 that is similar to unwind info formats on other platforms.

I would try to stay with 1, at least for now. The Linux x86 exception handling changes affect number of sub-systems in major way. If you start redoing yet another non-trivial part of the system as part of it, it will make it even harder to land.